gnovm/cmd/gno/util.go

```diff
@@ -100,6 +100,64 @@ func gnoPackagesFromArgs(args []string) ([]string, error) {
 	return paths, nil
 }
 
+func gnoPackagePathsFromPattern(patterns []string) ([]string, error) {
+	paths := []string{}
+	recursiveLookup := false
```

Comment:
Why is this flag outside the loop, since you never reset it within the loop? (once set)

Reply:
Fixed it: d23e55c68dcaa9ad33c0aa13b40bd707a0e17509